osquery/tables/system/posix/python_packages.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <string>
+#include <fstream>
+
+#include <boost/algorithm/string/trim.hpp>
+
+#include <osquery/core/conversions.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+
+namespace osquery {
+namespace tables {
+
+const int kNumFields = 2;
+const std::set<std::string> kPythonPath = {
+  "/usr/local/lib/python2.7/", "/usr/lib/python2.7/dist-packages",
+};
+
+
+// checks if given set of paths are sub-path of one another
+int checkOverlap() {
```

Comment:
After you've made the above change, can we nix this function? It's useful as the code currently stands, but with the change above it might lead us to unnecessary work.